nncf/experimental/common/graph/netron.py

```diff
@@ -106,10 +161,39 @@ def as_xml_element(self) -> ET.Element:
 GET_ATTRIBUTES_FN_TYPE = Callable[[NNCFNode], Dict[str, str]]
 
 
-# TODO(andrey-churkin): Add support for `PortDesc.precision` param.
+def convert_nncf_dtype_to_ov_dtype(dtype: Dtype) -> str:
+    """
+    Converts a nncf dtype to an openvino dtype string.
+
+    :param dtype: The data type to be converted. Should be one of the nncf Dtype.
```

Comment:
We can understand that from type hint.
```suggestion
    :param dtype: The data type to be converted.
```